README.md

```diff
@@ -66,7 +66,18 @@ Begin by [importing the repository](https://github.com/new/import) into a global
   - Privacy: Public
 - Press "Begin Import"
 
-The repository contains a GitHub Action that will run and update from upstream at least once per hour, see `./.github/workflows/update-azure-devops-templates-from-upstream.yaml`.
+#### Keeping the repository up to date
+
+The repository contains a GitHub Action that will automatically run and update the `main` branch from upstream at least once per hour, see [`./.github/workflows/update-azure-devops-templates-from-upstream.yaml`](https://github.com/XenitAB/azure-devops-templates/blob/main/.github/workflows/update-azure-devops-templates-from-upstream.yaml). It is not required to use this (you can keep the `main` branch up to date manually if you prefer), but it is recommended.
+
+In order for this to run correctly, you need to register a new (or reuse an existing) GitHub App **private to your organization** with the appropriate access. To do this, [go here](https://github.com/settings/apps/new). It is important that the app has read and write access to the `workflows`.
+
+As you register your app, you will receive an _application id_ and a _private key_. These need to be added to the secrets for your repository:
+
+- Add the _application id_ as a secret named `UPDATE_FROM_UPSTREAM_APP_ID` (this will look something like `152762`)
+- Add the _private key_ as a secret named `UPDATE_FROM_UPSTREAM_PRIVATE_KEY` (this is the contents of the `.pem` file , which starts with `-----BEGIN RSA PRIVATE KEY-----` and runs over several lines)
+
+add a secret to your clone called `UPDATE_FROM_UPSTREAM_PAT`. This secret needs to be a Personal Access Token (PAT) with access to the `workflow` scope. For information on how to create a PAT [see here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). For more information about how to add a secret, [see here](https://docs.github.com/en/actions/security-guides/encrypted-secrets).
```

Comment:
This should probably be removed, I assume we don't need a PAT as well as a github app